experiments/memcached-cat/main.go

```diff
@@ -100,8 +99,16 @@ func main() {
 	}
 
 	// Discover local CPU topology.
+	hpCores := topology.HpRangeFlag.Value()
 	beCores := topology.BeRangeFlag.Value()
+
+	if len(beCores) == 0 || len(hpCores) == 0 {
+		logrus.Errorf("HP & BE workloads ranges required! (please specify -experiment_hp_workload_cpu_range and -experiment_be_workload_l3_cpu_range)")
```

Comment:
You could have used flag.Name() so that you will be safe when flag name changes.